ansible/roles/opencontrail-controller/tasks/config.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Install templates
+  get_url:
+    url=https://raw.githubusercontent.com/Juniper/contrail-kubernetes/master/cluster/{{item}}
```

Comment:
Should these manifests also be committed to the ansible role itself?

If they are changed outside of this repository this is the first instance where ansible is going outside of the kubernetes repo. Seems like triaging any issue around this might be troublesome later on.